packages/web/app/views/patients/medication/MarTable.jsx

```diff
@@ -0,0 +1,271 @@
+`;
+
+const HeadingCell = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 12px;
+  font-size: 16px;
+  font-weight: 500;
+  color: ${Colors.darkestText};
+  border-top: 1px solid ${Colors.outline};
+  border-left: 1px solid ${Colors.outline};
+  height: 100%;
+`;
+
+const TimeSlotHeaderContainer = styled.div`
+  padding: 24px 0px 10px 0px;
+  height: 105px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-top: 1px solid ${Colors.outline};
+  border-left: 1px solid ${Colors.outline};
+  ${props => props.isCurrentTimeSlot && `background: #EBF0F5; color: ${Colors.primary};`}
+`;
+
+const TimeSlotText = styled.div`
+  font-weight: 400;
+  font-size: 12px;
+  transform: rotate(-90deg);
+  white-space: nowrap;
+  color: ${Colors.midText};
+`;
+
+const TimeSlotLabel = styled.div`
+  text-transform: capitalize;
+  color: ${Colors.darkestText};
+`;
+
+const SubHeader = styled.div`
+  font-size: 14px;
+  font-weight: 500;
+  color: ${Colors.midText};
+  padding: 10px 8px;
+  border-top: 1px solid ${Colors.outline};
+  border-left: 1px solid ${Colors.outline};
+  border-bottom: 1px solid ${Colors.outline};
+  margin-top: -1px;
+  grid-column: 1 / -1;
+  position: sticky;
+  top: 0;
+  z-index: 5;
+  background-color: ${Colors.white};
+`;
+
+const EmptyMessage = styled.div`
+  color: ${Colors.darkestText};
+  font-size: 14px;
+  font-weight: 500;
+  padding: 10px 8px;
+  border-top: 1px solid ${Colors.outline};
+  border-left: 1px solid ${Colors.outline};
+  grid-column: 1 / -1;
+`;
+
+const MedicationCellContainer = styled.div`
+  padding: 8px 12px;
+  font-size: 14px;
+  border-top: 1px solid ${Colors.outline};
+  border-left: 1px solid ${Colors.outline};
+  ${props => props.discontinued && `text-decoration: line-through;`}
+`;
+
+const StatusCell = styled.div`
+  border-top: 1px solid ${Colors.outline};
+  border-left: 1px solid ${Colors.outline};
+`;
+
+const MedicationGrid = styled.div`
+  display: grid;
+  grid-template-columns: minmax(100px, 1fr) repeat(12, 48px);
+`;
+
+const CurrentTimeOverlay = styled.div`
+  position: absolute;
+  top: 0;
+  width: 48px;
+  height: 100%;
+  z-index: 11;
+  right: ${p => (p.$length - p.$index - 1) * 48}px;
+  border: 1px solid ${Colors.primary};
+`;
+
+const formatTime = time => {
+  return format(parse(time, 'HH:mm', new Date()), 'ha').toLowerCase();
+};
+
+const MedicationCell = ({
+  id,
+  isPrn,
+  doseAmount,
+  units,
+  frequency,
+  route,
+  notes,
+  medication,
+  discontinued,
+}) => {
+  const doseAmountDisplay = isPrn ? (
+    <TranslatedText stringId="medication.table.variable" fallback="Variable" />
+  ) : (
+    doseAmount
+  );
+  return (
+    <React.Fragment key={id}>
+      <MedicationCellContainer discontinued={discontinued}>
+        <Box fontWeight={500}>{medication.name}</Box>
+        <Box>
+          <span>{doseAmountDisplay}</span>
+          <span>{` ${units}, ${frequency}, ${DRUG_ROUTE_LABELS[route]}`}</span>
```

Comment:
```suggestion
          <span> {units}, {frequency}, {DRUG_ROUTE_LABELS[route]</span>
```
This could be simplified